nuto/mechanics/mesh/GeometryMeshFem.cpp

```diff
@@ -0,0 +1,55 @@
+#include "nuto/mechanics/mesh/GeometryMeshFem.h"
+
+#include <sstream>
+#include "nuto/base/Exception.h"
+
+using namespace NuTo;
+
+InterpolationSimple& GeometryMeshFem::CreateInterpolation(const InterpolationSimple& interpolation)
+{
+    mInterpolations.push_back(interpolation.Clone());
+    return *mInterpolations.rbegin()->get();
+}
+
+CoordinateNode& GeometryMeshFem::NodeAtCoordinate(Eigen::VectorXd coords, double tol /* = 1.e-10 */)
+{
+    for (auto& node : this->CoordinateNodes)
+    {
+        Eigen::VectorXd globalNodeCoords = node.GetCoordinates();
+        if ((globalNodeCoords - coords).isMuchSmallerThan(tol, 1))
+            return node;
+    }
+    std::stringstream coordsString;
+    coordsString << coords.transpose();
+    throw NuTo::Exception(__PRETTY_FUNCTION__, "There is no coordinate node at " + coordsString.str());
```

Comment:
Why not
~~~cpp
std::stringstream message;
message << "There is no coordinate node at " << coords.transpose() << ".";
throw NuTo::Exception(__PRETTY_FUNCTION__, message.str());
~~~
?